cutil/kubeconfig/kubeconfig.go

```diff
@@ -32,7 +33,14 @@ func GetConfig(existing *cluster.Cluster) error {
 	if err != nil {
 		return err
 	}
-	signer, err := ssh.ParsePrivateKey(pemBytes)
+
```

Comment:
Can we have the code try it without a passphrase first? Then if we get a passphrase err prompt for one? 

Also we will need a way to disable all together via an environmental variable. User's will be wrapping the apply in automation scripts, and we can't leave them hanging (pun intended)